### DISCUSSION TITLE
[Action Required - Final Notice] Upgrade Amazon Aurora Serverless v1 PostgreSQL-compatible edition 10.x by January 31, 2023

### DISCUSSION THREAD

**Discussion by Aaron-E-Walsh** [mozilla/hubs#5892] [Hubs Cloud Questions]
Hi all, we have received the "final notice" from AWS requiring us to update to Aurora PostgreSQL 11 for Amazon Aurora Serverless v1 by January 31. 

Has anyone done this for their own Hubs Cloud installation on AWS, and if so did you have any problems or issues we should be aware of?

Naturally I'm concerned about the upgrade breaking or disrupting our instance of Hubs, and so your advice and insight on the upgrade are greatly appreciated.

With thanks,
Aaron

**Comment by pattersonbl2** [comment]
We have released a new template (version 1.1.5) that will update the database engine.  I have updated my instance without any issues. Even if a person doesn't update the database engine, AWS will update it for you and it shouldn't cause any problems.

**Reply by Aaron-E-Walsh** [reply]
Thank you. We are going to update our template in a few days and try to upgrade the database as well so that it doesn't happen automatically (via automatic AWS upgrade)  at a time that we might need the system to be online. Thanks again.
